docs: Improve Actions docs. (#51145)
vercel/next.js: docs/02-app/01-building-your-application/03-data-fetching/04-server-actions.mdx

### Actions

Actions are an experimental feature in React, allowing you to run `async` code in response to a user interaction.

> Actions are not Next.js or React Server Components specific, however, they are not yet available in the stable version of React. When using Actions through Next.js, you are opting into using the React `experimental` channel.

Reviewer: I think you can drop the however here. I also find this paragraph confusing. Does this wording maintain the intended meaning?
```suggestion
Actions are not Next.js or React Server Components specific, and unavailable in the stable version of React. When using Actions through Next.js, you are opting into using the React `experimental` channel.
```

Author: The "not yet" is intentionally worded, since they have landed and are available to use _here_, but not yet for stable React.
